benchmarks/unit_benchmarks.py

```diff
@@ -0,0 +1,98 @@
+import pybamm
+import numpy as np
+
+R = 1e-5
+
+
+def time_create_expression():
```

Comment:
You should define a class like with the [other benchmarks](https://github.com/pybamm-team/PyBaMM/blob/develop/benchmarks/time_solve_models.py). The first bit can be pushed under the `setup` method, and then have the different benchmarks.